### DISCUSSION TITLE
pag文件 在viewer中看到的特效 在web端展示不全

### DISCUSSION THREAD

**Discussion by zxcv66666** [Tencent/libpag#1723] [Q&A] [answered]
pag 导出的文件中 包含 3D旋转和一些滑动特效
<img width="359" alt="image" src="https://github.com/Tencent/libpag/assets/144442760/a763951a-9290-4401-b47e-d99d03535154">

当我把它放到web端展示 就只剩下一部分的特效可以展示出来 旋转和滑动都消失了，使用的sdk版本为4.3.3
<img width="297" alt="image" src="https://github.com/Tencent/libpag/assets/144442760/7a76a5cb-8323-4cb9-95de-8b0365cb290d">
想问一下是我展示的问题 还是导出的pag文件有问题
下面附上pag文件 感谢指导
[测试1.pag.zip](https://github.com/Tencent/libpag/files/12573880/1.pag.zip)

**Comment by zhalice2011** [comment, marked as answer]
您好, 3D特性是企业版的功能, 如果您有需要可以联系我们商务

https://pag.art/feature.html#comparison
